packages/L10n/src/locales/en.yml

```diff
@@ -36,6 +36,8 @@ en:
         cancelled: Cancelled
         complete: Complete
         idle: Idle
+        file_progress: "{{name}} is {{progress}}"
+        uploading_percent: Uploading of {{a11yProgress}}%
```

Comment:
@mikrotron could you review these translations?

Reply:
Needs a bit of work.  I am displaying the `aria-label` in blue in the screenshot below so you can see them all at a glance.  Just needs a bit more finesse.  

<img width="618" alt="Screen Shot 2021-06-07 at 5 21 10 PM" src="https://user-images.githubusercontent.com/14944896/121103638-31983a00-c7b5-11eb-94b7-11911a684ada.png">


As you can see, the actual progress gets announced by the screenreader since the list item is a `role=progressbar`, so the "of 25% can probably just get dropped from the `aria-label`.

Reply:
Right, I realized that too :/, but I needed that "25%" so aria-live didn't announce every percentage increase from 0-100. I think I might have another workaround, where the progressbar doesn't get the 25% increments. I'll take another stab at it!

Reply:
Getting so close! The wording for the `Uploading` status is still just a little bit off though:
<img width="654" alt="Screen Shot 2021-06-08 at 10 42 49 PM" src="https://user-images.githubusercontent.com/14944896/121299962-01cb5e00-c8ab-11eb-8039-1472fd606820.png">

Reply:
I think just `Filename.ext is uploading, 25%` might be sufficient. What do you think?